debian/copyright

```diff
@@ -1,7 +1,11 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
 Files: *
-Copyright: 2021 KUNBUS GmbH
+Copyright: 2021-2022 KUNBUS GmbH
```

Comment:
Why do the dates differ?

Reply:
We did a commit in 2022 so update the software and did not update this while the last release of the software.